DQMOffline/Trigger/plugins/BPHMonitor.cc

```diff
@@ -1433,11 +1430,10 @@ double BPHMonitor::Prescale(const std::string hltpath1,
   if (flag && (theSame_num.size() == theSame_den.size())) {
     L1P = 1;  //den and num have the same set of l1 seeds
   } else {
-    if (!(hltPrescale_->prescaleValuesInDetail(iEvent, iSetup, hltpath1)).first.empty()) {
+    if (not prescales_num.first.empty()) {
       Prescale_num = 1;
-      for (size_t iSeed = 0; iSeed < (hltPrescale_->prescaleValuesInDetail(iEvent, iSetup, hltpath1)).first.size();
-           ++iSeed) {
-        int l1 = (hltPrescale_->prescaleValuesInDetail(iEvent, iSetup, hltpath1)).first.at(iSeed).second;
+      for (size_t iSeed = 0; iSeed < prescales_num.first.size(); ++iSeed) {
+        auto l1 = prescales_num.first.at(iSeed).second;
         if (l1 < 1)
           continue;
         if (l1 == 1) {
```

Comment:
Is this ok with rounding effects?

Reply:
See https://github.com/cms-sw/cmssw/pull/39220#discussion_r975393631.